lib/grpc/server/transcode.ex

```diff
@@ -0,0 +1,80 @@
+defmodule GRPC.Server.Transcode do
+  @moduledoc false
+  alias GRPC.Server.Router.{Query, FieldPath}
```

Comment:
```suggestion
  alias GRPC.Server.Router.Query
  alias GRPC.Server.Router.FieldPath
```

note: I prefer separating aliases because it makes it a lot easier to do project-wide searches for a module